src/streamer/jamstreamer.cpp

```diff
@@ -0,0 +1,37 @@
+#ifndef _WIN32
+#include "jamstreamer.h"
+
+using namespace streamer;
+
+// --- CONSTRUCTOR ---
+CJamStreamer::CJamStreamer() {}
+
+// --- PROCESS ---
+// put the received pcm data into the pipe to ffmpeg
+void CJamStreamer::process( int iServerFrameSizeSamples, CVector<int16_t> data ) {
```

Comment:
`data` should be a reference to avoid copying all its data per function call: `CVector<int16_t>& data`

Reply:
i can't get this one to work though. is it in any way possible for you to fork my code and raise a pull request? (that whole pointer business drives me mad, i'm still not a programmer)

Reply:
Seems like it worked: https://github.com/dingodoppelt/jamulus/pull/2